.husky/post-merge

```diff
@@ -4,7 +4,3 @@
 
 yarn install
 yarn build
-
-# React's build doesn't pick up on patch changes, so we need to run it twice
-yarn patch-package
-yarn react build
```

Comment:
I know you already merged this, but man this change feels so good. I no longer have to grab a coffee post-merge. Maybe it's a bad thing too... 👿

Reply:
Ooh, get ready for more @wlee221 💪